<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectForm</class>
 <widget class="QWidget" name="NewProjectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	text-align: center;
	font: 57 16pt &quot;Sono Medium&quot;;
}

QWidget {
    background-color: rgb(208, 225, 249);
}

QPushButton {
    background-color: rgb(77, 100, 141);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QPushButton:focus {
    background-color: rgb(40, 54, 85);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QLineEdit {
	text-align: centrel;
    border: none;
	border-radius: 5px;
    background-color: #E6F4FF;
	font: 57 16pt &quot;Sono Medium&quot;;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QPushButton" name="back_button">
     <property name="styleSheet">
      <string notr="true">QLabel {
	text-align: center;
	font: 57 16pt &quot;Sono Medium&quot;;
}

QWidget {
    background-color: rgb(208, 225, 249);
}

QPushButton {
    background-color: rgb(77, 100, 141);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QPushButton:focus {
    background-color: rgb(40, 54, 85);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QPushButton[class=&quot;CustomSpinboxButton&quot;] {
    background-color: rgb(77, 100, 141);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QPushButton[class=&quot;CustomSpinboxButtonPressed&quot;] {
    background-color: rgb(40, 54, 85);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QDoubleSpinBox {
	text-align: centrel;
    border: none;
	border-radius: 5px;
    background-color: #E6F4FF;
	font: 57 16pt &quot;Sono Medium&quot;;
}

QDoubleSpinBox:focus {
    border: 2px solid #0057D9;
    background-color: #E6F4FF;
}</string>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="error_message_label">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Название проекта</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Путь до проекта</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="project_name_input">
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="project_path_input">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="choose_path_button">
     <property name="text">
      <string>Выбрать путь</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="create_new_project_button">
     <property name="text">
      <string>Создать проект</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
